tests/model_explainability/trustyai_service/utils.py

```diff
@@ -68,3 +75,69 @@ def _get_mariadb_pods() -> list[Pod]:
             condition=Pod.Condition.READY,
             status="True",
         )
+
+
+@retry(wait_timeout=Timeout.TIMEOUT_2MIN, sleep=5)
+def validate_trustyai_service_db_conn_failure(
+    client: DynamicClient, namespace: str, label_selector: Optional[str]
+) -> bool:
+    """
+    Waits for TrustyAIService pod to fail and checks if the pod is in a CrashLoopBackOff state and
+    the LastState is in terminated state and the cause was a MariaDB TLS certificate exception.
+    Also checks if there are more than one pod for the service.
+
+    Args:
+        client: DynamicClient
+        namespace: Namespace
+        label_selector: Optional(str)
+
+    Returns: bool
+    """
+    pods = list(Pod.get(dyn_client=client, namespace=namespace, label_selector=label_selector))
+    mariadb_conn_failure_regex = (
+        r"^.+ERROR.+Could\snot\sconnect\sto\smariadb:.+\sPKIX\spath\svalidation\sfailed:\s"
+        r"java\.security.cert.CertPathValidatorException:\ssignature\scheck\sfailed"
```

Comment:
are the explicit `\s` needed or can you just use whitespace?
e.g 
`Could\snot\sconnect` -> `Could not connect`